packages/graph/hash_graph/lib/graph/src/store/postgres/query/expression/join_clause.rs

```diff
@@ -11,6 +13,39 @@ pub struct JoinExpression<'q> {
 impl<'q> JoinExpression<'q> {
     #[must_use]
     pub const fn from_tables(table: Table, on: Table) -> Self {
```

Comment:
I might have missed this previously, but what is `table` vs `on` here? Is `table` the source and `on` the thing we're joining on?

Reply:
I renamed these in another PR to `join` and `on`. It's joining the `join` table and compares the ids with `on`.